### PR TITLE
PTA changes

### PR DESCRIPTION
**Pull request by pranchalsomani**

### CODE REVIEW

File: Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md [hunk 1]
```diff
@@ -51,5 +51,9 @@
     7. User needs to delete the PIM non SC-ALT role assignment(s) and migrate user's critical role assignment(s) to SC-ALT PIM after the script 
        execution.
 
+## FAQs:
+     Q.1. If i am getting this error "Unable to find type [Microsoft.Azure.Commands.Common.Authentication.AzureSession]" what i need to do ?
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md:55]: Remove 'If'

Comment (TarunKrShukla) [reply Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md:55]: Or something like this,
While running the remediation script '<ScriptName>' I am getting error "Unable to find type [Microsoft.Azure.Commands.Common.Authentication.AzureSession]". How can I resolve this error?

File: Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md [hunk 2]
```diff
@@ -51,5 +51,9 @@
...
+     A.1 Please run this command "Install-Module -Name Az.Accounts -Scope CurrentUser -Force -Repository "PSGallery" -Force  -AllowClobber -ErrorAction Stop" before loading the script.
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md:56]: In a fresh session.

File: Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md (hunk 1 above)
Comment (pranchalsomani) [reply Scripts/RemediationScripts/Migrate-PermanentAndNonSCALTPrivilegedToSCALTPrivilegedRoleAssignments/Readme.md:55]: This file is specific to this script only so no need to mention the script name.

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**